samples/subsys/display/lvgl/boards/mimxrt1060_evk_mimxrt1062_qspi_C.conf

```diff
@@ -0,0 +1,9 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+# Enable PXP DMA engine and set rotation angle to 0 degrees.
+# This allows us to verify the DMA driver functions without altering
+# the output image
```

Comment:
the comment here also seems to imply this conf is really only here for testing purposes? ("verify the DMA driver"??). This doesn't sound right. Also, maybe CONFIG_MCUX_ELCDIF_PXP should simply default to Y at the board/soc level when DMA is enabled?

Reply:
this is reused from former mimxrt1060_evk board,  CONFIG_MCUX_ELCDIF_PXP consumers a lot, we do not want it enabled by default if user do not use display.

Reply:
Not enabling it by default is okay, but the comment needs to be changed.
I'd honestly just drop it - why would you enable DMA and PXP related settings if not to use/test them?